setup.py

```diff
@@ -26,11 +26,42 @@
         '\nuse it. Set WALLY_ABI_PY_WHEEL_USE_LIB=static to link with the static library.\n',
         file=sys.stderr)
 
+def str_to_bool(str, if_none=ValueError, if_invalid=ValueError):
+    if str is None:
+        default = if_none
+    else:
+        lstr = str.lower()
+        if lstr in ('0', 'n', 'no', 'f', 'false', 'off'):
+            return False
+        elif lstr in ('1', 'y', 'yes', 't', 'true', 'on'):
+            return True
+        default = if_invalid
+    if isinstance(default, BaseException):
+        raise default('Invalid flag value: {}'.format(str))
+    return default
+
+def pkg_default(pkg, if_true, if_false=None):
+    b = str_to_bool(pkg, None, None)
+    return if_true if b == True else if_false if b == False else pkg
+
+PKG_SECP256K1 = pkg_default(
+    os.environ.get('WALLY_ABI_PY_WHEEL_USE_PKG_SECP256K1'), 'libsecp256k1_zkp')
+
+if PKG_SECP256K1 is not None:
+    try:
+        import pkgconfig
+    except ImportError:
+        print('Error: You specified WALLY_ABI_PY_WHEEL_USE_PKG_SECP256K1, but you do not have'
+            '\nthe pkgconfig Python module installed.',
```

Comment:
nit: `print('Error: WALLY_ABI_PY_WHEEL_USE_PKG_SECP256K1 requires the pkgconfig Python module.')`

Reply:
That's longer than 80 characters, but I'll format it without a line break as you have requested. Should I also remove the line wrapping from the other error/warning messages?

Reply:
I'm fine with the other errors, and for python lines up to 120 characters are fine, cheers.